packages/plugin-ocr-convert/src/server/actions/index.ts

```
@@ -0,0 +1,19 @@
+import Application from '@tachybase/server';
+
+import * as fileConvert from './fileConvert';
+import * as ocrConvert from './ocrConvert';
+
+function make(name, mod) {
+  return Object.keys(mod).reduce(
+    (result, key) => ({
+      ...result,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)